contracts/SimpleYieldEulerLever.sol

```diff
@@ -0,0 +1,395 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.14;
+
+import "@yield-protocol/utils-v2/contracts/cast/CastU128I128.sol";
+import "@yield-protocol/utils-v2/contracts/cast/CastI128U128.sol";
+import "@yield-protocol/utils-v2/contracts/cast/CastU256U128.sol";
+import "@yield-protocol/utils-v2/contracts/cast/CastU256I256.sol";
+import "@yield-protocol/utils-v2/contracts/token/TransferHelper.sol";
+import "./interfaces/IEulerMarkets.sol";
+import "./interfaces/IEulerEToken.sol";
+import "./YieldLeverBase.sol";
+
+/// @title A simple euler lever designed to work for one euler token & its underlying at a time
+/// @author iamsahu
+/// @notice Working:
+///         - Get flash loan of USDC/DAI/WETH
+///         - Deposit to get eulerToken
+///         - Deposit & borrow against it
+///         - Sell the fyToken to get USDC/DAI/WETH
+///         - Close the flash loan
+contract SimpleYieldEulerLever is YieldLeverBase {
```

Comment:
```suggestion
contract YieldEulerLever is YieldLeverBase {
```
If you want to keep the more complex YieldEulerLever please give it another name and move it inside `contracts/draft`